publish-release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+TAG=$1
+if [[ -z "$TAG" ]]; then
+  echo "Expected release tag to be passed as the first argument"
+  exit 1
+fi
+
+if [[ ! "$TAG" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "Invalid release tag: $TAG"
+  exit 1
+fi
+
+echo "Generating grpcwebproxy binaries"
+cd go
+./generate-grpcwebproxy-binaries.sh
+ls -l ./build
+cd ..
+
+echo "Publishing $TAG"
+
+# Create github release and attach server binaries
+./node_modules/.bin/github-release upload \
```

Comment:
Are you referring to `github-release`? I've added it to the project's list of npm dependencies.

Reply:
Yeah, I just don't understand how travis works, I don't think we're checking in `./node_modules/`, does travis guarantee that it will run `npm install` before any script or something?

Reply:
I believe so, however given our current flakey tests I am concerned that the publish step will never be reached as the test test that proceeds it will likely fail. 

You mentioned using Github Actions as an alternative - these are new to me, let me take a look.